### ISSUE TITLE
[Package Issue]: Microsoft.Sysinternals.TCPView

### ISSUE THREAD

**Issue by stym-singh** [microsoft/winget-pkgs#101615]
### Please confirm these before moving forward

- [X] I have searched for my issue and not found a work-in-progress/duplicate/resolved issue.
- [X] I have not been informed if the issue is resolved in a preview version of the winget client.

### Category of the issue

Installer hash mismatch.

### Brief description of your issue

TCPView version 4.17 installer hash does not match.

### Steps to reproduce

`winget install tcpview`

### Actual behavior

The command should successfully install TCPView version 4.17

### Expected behavior

The application installation should complete without any hash mismatch related errors.

### Environment

```raw
Windows Package Manager v1.4.10173
Copyright (c) Microsoft Corporation. All rights reserved.

Windows: Windows.Desktop v10.0.22621.1413
System Architecture: X64
Package: Microsoft.DesktopAppInstaller v1.19.10173.0
```

**Comment by BrandonWanHuanSheng** [comment]
Get metadata from the application. It look like I do not need a full URL. I use my application called wget to do this.  At the same time, I verified another version.

**Comment by BrandonWanHuanSheng** [comment]
It look like Application-Metadata in tcpview\.rsrc\version.txt That one is used by wingetcreate to payload the file.

**Comment by BrandonWanHuanSheng** [comment]
Did you saw 2 commit?

**Comment by mdanish-kh** [comment]
Close with reason: Resolved in #101799;
